pages/docs/submissions.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Submissions
+description: Submitting your Web Integration
+---
+
+# {% $markdoc.frontmatter.title %}
+
+## What web integrations can be submitted?
+
+We welcome and appreciate all web integrations that exhibit the community standards at NZXT and follow our [terms of service](https://nzxt.com/legal/terms-of-service) and [privacy policy](https://nzxt.com/legal/privacy-policy). At this time, we can only accept web integrations that can be statically hosted. So if your web integration requires a server (e.g. due to OAuth requirements), we encourage you to open a PR and submit to our [community submissions](https://github.com/NZXTCorp/web-integrations-examples/blob/main/community.md) list instead.
+
+## Submitting your web integration
+
+Got your web integration working and publicly accessible on the internet? Ready to share your creation to the NZXT community? Please send an email to [web.integrations@nzxt.com](mailto:web.integrations@nzxt.com) and include the following details:
```

Comment:
What should we verify each of the requests?

Reply:
@RobbiePlata just to reduce some spam can you not use a markdown hyperlink & just have it read "web [dot] integrations [at] nzxt [dot] com"